h/sentry.py

```diff
@@ -14,6 +14,11 @@
 
 from h import __version__
 
+PROCESSORS = (
+    'raven.processors.SanitizePasswordsProcessor',
+    'raven.processors.RemovePostDataProcessor',
```

Comment:
I'd suggest that it would probably be a good idea to add a comment to indicate why we are removing POST data here, given that the issue with multipart forms is non-obvious.